dask/array/routines.py

```diff
@@ -786,7 +787,14 @@ def _isin_kernel(element, test_elements, assume_unique=False):
     return values.reshape(element.shape + (1,) * test_elements.ndim)
 
 
-@wraps(getattr(np, 'isin', None))
+def safe_wraps(wrapped, assigned=functools.WRAPPER_ASSIGNMENTS):
+    if all(hasattr(wrapped, attr) for attr in assigned):
+        return wraps(wrapped, assigned=assigned)
+    else:
+        return lambda x: x
```

Comment:
Any thoughts on moving this to a `utils` or similar module?

Reply:
done